projects/gateway/pkg/syncer/setup_syncer.go

```diff
@@ -269,6 +272,26 @@ func RunGateway(opts translator.Opts) error {
 
 	validationServerErr := make(chan error, 1)
 	if opts.Validation != nil {
+		// make sure non-empty WatchNamespaces contains the gloo instance's own namespace if
+		// ReadGatewaysFromAllNamespaces is false
+		controllerNamespace := os.Getenv(setuputils.PodNamespace)
```

Comment:
i know it was originally my suggestion, but i think we should be able to do this instead (and thus remove the check following)
```suggestion
settings.Metadata.Namespace
```
as this should always equal `POD_NAMESPACE` (or the equivalent if passed by flag in CLI) -- even in the case of running gloo outside of k8s

Reply:
this is going to require some refactoring - the settings variable isn't passed into the useGateway function. This function  has one other usage, and I don't immediately see how to get the namespace in the other case (gateway.go)